robottelo/ui/locators.py

```diff
@@ -1383,6 +1383,15 @@ def __iter__(self):
     "repo.fetch_upstream": (
         By.XPATH, ("//form[@bst-edit-text='repository.docker_upstream_name']"
                    "/div[@class='bst-edit']/div/span[2]")),
+    "repo.fetch_packages": (
+        By.XPATH, "//td[span[text()='Packages']]/following-sibling::td",
```

Comment:
For my case, it was really impossible to address proper DOM element without reference to really dynamic things. Here we have good enough locators that will work just nice for our application. Yes, that is not best idea to access element by text, but we have huge amount of locators that work in the same way. Don't think that we need to make current case so unique and spend time on fixing Foreman application itself (create bug, find solution, fix the code, ...).
@omaciel Please, let us know if we need do that anyway